services/emailService.go

```diff
@@ -85,5 +85,12 @@ func (s *emailService) SendEmailVerificationEmail(user entities.User, emailToken
 }
 
 func (s *emailService) SendPasswordResetEmail(user entities.User, emailToken string) error {
-	return nil
+	return s.SendEmail(
```

Comment:
If we wanted to update the design of the email, would we change it here?

Reply:
Yeah, the `EmailService` is responsible for compiling the email. The 2nd and 3rd arguments to `SendEmail` are the email's html and raw text body.